resources/zib/zib-OutcomeOfCare.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-OutcomeOfCare" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-OutcomeOfCare" />
+  <name value="ZibOutcomeOfCare" />
+  <title value="zib OutcomeOfCare" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="The OutcomeOfCare concept describes the determined status of the patient in terms of a certain problem. Comparing the healthcare result with the treatment objective provides insight into the effectively of the nursing interventions/activities carried out for this problem. It is a part of the nursing process, together with nursing diagnoses/problems, the treatment objective and nursing interventions." />
+  <purpose value="This DiagnosticReport resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) OutcomeOfCare -v3.2(2020EN)](https://zibs.nl/wiki/OutcomeOfCare-v3.2(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-outcomeofcare-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/OutcomeOfCare-v3.2(2020EN)" />
+    <name value="zib OutcomeOfCare-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="DiagnosticReport" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/DiagnosticReport" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="DiagnosticReport">
```

Comment:
A comment related to the reversed reference to Intervention should be added here, in line with the profiling guidelines.